plugins/base/src/main/resources/dokka/scripts/symbol-parameters-wrapper_deferred.js

```diff
@@ -13,6 +13,17 @@ const wrapSymbolParameters = (symbol) => {
     }
 
     let symbolBlockWidth = symbol.clientWidth
+
+    // Even though the script is marked as `defer` and we wait for `DOMContentLoaded` event,
+    // it can happen that `symbolBlockWidth` is 0, indicating that something hasn't been loaded.
+    // Re-try after some time, should work. Should not go into infinite recursion because
```

Comment:
Have you tried [to force restyle](https://stackoverflow.com/questions/18564942/clean-way-to-programmatically-use-css-transitions-from-js/31862081#31862081)?

Reply:
I did try using `getComputedStyle(elem)` when I started implementing this change and came across similar problems, but it didn't work (returned all 0s), so I had to make the script `defer`, which helped. Haven't tried it in this scenario as it's only reproducible if hosted on GitHub pages for some reason (and not always).

Anyhow, relying on undocumented behaviour from 2012 seems shady, `setTimeout` for 100 millis should be more than enough and should work in all browsers.

But if you propose a specific code suggestion, we can try. Will have to test it in all browsers and pray to god nothing changes though.